MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -6851,11 +6850,26 @@ public int getAtBUnitRatingMod() {
                 : reputation.getAtbModifier();
     }
 
+    /**
+     * Retrieves the unit rating modifier as described in Campaign Operations. This value is equal
+     * to the total reputation score divided by ten, rounded down to the nearest whole number.
+     *
+     * @return The unit rating modifier as described in Campaign Operations.
+     */
+    public int getCamOpsUnitRatingMod() {
```

Comment:
Is there value in moving this into the Reputation class - cooking dinner so the class might be called something else? Call the new method something like `getReputationModifier()`.

Reply:
I did actually try to do that first, but there is already a different getReputationModifier() there that does something based on the unit experience levels instead. I wasn't sure how that was used and so I didn't want to mess with it--could you look at that when you get a minute and let me know what you think?

Thanks!

Reply:
I’ll take a look once I’ve had dinner.